<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Yunus Emre Akgül</author>
 <class>mainWindow</class>
 <widget class="QWidget" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Data Logger</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>2</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="homePage">
    <widget class="QPushButton" name="userButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>512</width>
       <height>300</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>512</width>
       <height>300</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border: none;
background: rgb(15, 188, 249);
}

QPushButton:hover {
background: rgb(75, 207, 250);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/icons/svg/user.svg</normaloff>:/icons/svg/user.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="graphButton">
     <property name="geometry">
      <rect>
       <x>512</x>
       <y>300</y>
       <width>512</width>
       <height>300</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>512</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>512</width>
       <height>300</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border: none;
background: rgb(128, 142, 155);

}

QPushButton:hover {
background: rgb(210, 218, 226);

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/icons/svg/chart-histogram.svg</normaloff>:/icons/svg/chart-histogram.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="listButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>512</width>
       <height>300</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>512</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>512</width>
       <height>300</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border: none;
background: rgb(255, 63, 52);

}

QPushButton:hover {
background: rgb(255, 94, 87);

}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/icons/svg/document.svg</normaloff>:/icons/svg/document.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="settingsButton">
     <property name="geometry">
      <rect>
       <x>512</x>
       <y>0</y>
       <width>512</width>
       <height>300</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>512</width>
       <height>300</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>512</width>
       <height>300</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border: none;
background: rgb(0, 216, 214);
}

QPushButton:hover {
background: rgb(52, 231, 228);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/icons/svg/settings.svg</normaloff>:/icons/svg/settings.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="loginPage">
    <property name="styleSheet">
     <string notr="true">QWidget{
background: rgb(15, 188, 249);
}</string>
    </property>
    <widget class="QLineEdit" name="usernameText">
     <property name="geometry">
      <rect>
       <x>312</x>
       <y>300</y>
       <width>400</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Kullanıcı Adı</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userpasswordText">
     <property name="geometry">
      <rect>
       <x>312</x>
       <y>350</y>
       <width>400</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::PasswordEchoOnEdit</enum>
     </property>
     <property name="placeholderText">
      <string>Şifre</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>312</x>
       <y>400</y>
       <width>195</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border: none;
background: rgb(52, 231, 228);
}

QPushButton:hover {
background: rgb(0, 216, 214);
}</string>
     </property>
     <property name="text">
      <string>Giriş</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="newUserButton">
     <property name="geometry">
      <rect>
       <x>517</x>
       <y>400</y>
       <width>195</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border: none;
background: rgb(11, 232, 129);
}

QPushButton:hover {
background: rgb(5, 196, 107);
}</string>
     </property>
     <property name="text">
      <string>Şifremi unuttum</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="logoQgraph">
     <property name="geometry">
      <rect>
       <x>412</x>
       <y>75</y>
       <width>200</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginBackButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>90</width>
       <height>600</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/icons/svg/back-arrow.svg</normaloff>:/icons/svg/back-arrow.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="settingPage">
    <property name="styleSheet">
     <string notr="true">
QWidget {
background: rgb(0, 216, 214);
}

</string>
    </property>
    <widget class="QPushButton" name="settingsBackButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>90</width>
       <height>600</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/icons/svg/back-arrow.svg</normaloff>:/icons/svg/back-arrow.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="listPage">
    <property name="styleSheet">
     <string notr="true">
QWidget {
background: rgb(255, 63, 52);
}
</string>
    </property>
    <widget class="QPushButton" name="listBackButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>90</width>
       <height>600</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/icons/svg/back-arrow.svg</normaloff>:/icons/svg/back-arrow.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="graphPage">
    <property name="styleSheet">
     <string notr="true">
QWidget {
background: rgb(128, 142, 155);
}
</string>
    </property>
    <widget class="QPushButton" name="graphBackButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>90</width>
       <height>600</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/icons/svg/back-arrow.svg</normaloff>:/icons/svg/back-arrow.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icons/svg/home.svg</normaloff>:/icons/svg/home.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>2</number>
  </property>
  <property name="gridDeltaY">
   <number>2</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
